specification/resources/resource-manager/Microsoft.Authorization/stable/2020-05-01/privateLinks.json

```diff
@@ -335,13 +384,67 @@
         },
         "x-ms-examples": {
           "Delete a Resource Management Private Link.": {
-            "$ref": "./examples/ListResourceManagementPrivateLink.json"
+            "$ref": "./examples/ListSubscriptionResourceManagementPrivateLink.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Authorization/resourceManagementPrivateLinks": {
+      "get": {
+        "tags": [
+          "ResourceManagementPrivateLink"
+        ],
+        "operationId": "ResourceManagementPrivateLink_ListAtRGLevel",
+        "description": "Get all the resource management private links in a resource group.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group. The name is case insensitive.",
+            "minLength": 1,
+            "maxLength": 90
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Ok - Returns an array of private links.",
+            "schema": {
+              "$ref": "#/definitions/ResourceManagementPrivateLinkListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Delete a Resource Management Private Link.": {
+            "$ref": "./examples/ListResourceGroupResourceManagementPrivateLink.json"
           }
         }
       }
     }
   },
   "definitions": {
+    "PrivateLinkAssociationObject": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/PrivateLinkAssociationProperties",
+          "description": "The properties of the PrivateLinkAssociation."
+        }
+      }
+    },
```

Comment:
If this is only used for input, this is identical to `PrivateLinkAssociation` therefore we do not need to introduce this